resources/views/myhub/programme.blade.php

```diff
@@ -5,45 +5,32 @@
             <div class="px-6 py-6 bg-gray-50 dark:bg-gray-800">
                 <div class="flex flex-row justify-between">
                     <h2 class="font-semibold text-2xl text-gray-800 dark:text-gray-200 leading-tight">Programme</h2>
-
-                    <div>
-                        <a type="button"
-                           href="#"
-                           class="hidden lg:block md:block text-white bg-gradient-to-r from-cyan-400 via-cyan-500 to-cyan-600 hover:bg-gradient-to-br focus:ring-4 focus:outline-none focus:ring-cyan-300 dark:focus:ring-cyan-800 font-medium rounded-full text-sm px-5 py-2.5 text-center mr-2 mb-2">
-                            Do you want to host lecture/workshop?
-                        </a>
-                    </div>
                 </div>
 
                 @if($presentations->isEmpty())
                     <p class="text-l pt-16 text-gray-800 dark:text-gray-200">You need to register to a presentation or workshop first!</p>
+
+                    <a href="{{ route('my-programme-register') }}" type="button"
```

Comment:
Something seems to be up with the styling. The register button is right up against the previous line. Probably add some margin to the top of the register button.